manifests/k/KDE/kate/21.04.1/KDE.kate.locale.en-US.yaml

```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: KDE.kate
+PackageVersion: 21.04.1
+PackageLocale: en-US
+Publisher: KDE

```

Comment:
```suggestion
Publisher: KDE e.V.
```
![image](https://user-images.githubusercontent.com/11600822/121036326-95673680-c7a6-11eb-89ac-b0e594b679b8.png)

Reply:
Other Packages from KDE have only KDE as Publisher and not KDE e.V.

Reply:
Those will be fixed soon.